test/lib/browsers/base.ts

```diff
@@ -5,21 +5,18 @@ export type Event = 'request'
  * classes should build on, it is the bare
  * methods we aim to support across tests
  */
-export abstract class BrowserInterface implements PromiseLike<any> {
-  private promise?: Promise<any>
-  then: Promise<any>['then']
-  catch: Promise<any>['catch']
-  finally: Promise<any>['finally'];
```

Comment:
These three were the main culprit of the failed compilation. They are also incorrect because a `BrowserInterface` instance does not have these properties defined unless the return value from its `chain()` method is used.